### ISSUE TITLE
Error when searching for movie (possibly due to non-unicode characters)

### ISSUE THREAD

**Issue by toddrob99** [Waterboy1602/Addarr#23]
Running in docker, current master version. Search for `Friends مش (2010)` and the following error will be thrown.

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/telegram/ext/dispatcher.py", line 340, in process_update
    handler.handle_update(update, self, check, context)
  File "/usr/local/lib/python3.8/site-packages/telegram/ext/conversationhandler.py", line 465, in handle_update
    new_state = handler.handle_update(update, dispatcher, check_result, context)
  File "/usr/local/lib/python3.8/site-packages/telegram/ext/handler.py", line 119, in handle_update
    return self.callback(update, context)
  File "/usr/src/addarr/addarr.py", line 325, in searchSerieMovie
    chat_id=update.effective_message.chat_id, photo=output[position]["poster"]
IndexError: list index out of range
```

I found this while working on other changes, and haven't had a chance to trace the cause. The movie I found the issue with does have a poster URL (here's the escaped data from my log):

`{\'title\': \'Friends مش\', \'seasonCount\': 1, \'poster\': \'https://artworks.thetvdb.com/banners/posters/332606-1.jpg\', \'year\': 2010, \'id\': 332606}`

**Comment by Waterboy1602** [comment]
When I try to reproduce your search command, I don't get an error. I searched for "Friends مش (2010)" as a movie and series, both times no error message. Could you check if you still get the same error and at which specific moment you get the error?

**Comment by toddrob99** [comment]
Try searching for “Friends مش” — I think the first spot where I pasted in the description got messed up somehow. I just copied that from the log at the bottom of the description and searched for it, and the bot didn’t send the movie poster. I assume it threw the error but I’m not near the log right now.

**Comment by Waterboy1602** [comment]
I've searched for "Friends مش" as a movie and did indeed get the same result. It sends "Is this the movie you want to add", but with no poster. The reason for this is that Radarr didn't find any results. Same bug/problem as #26. I've made a commit that fixes this bug: [8ecd766](https://github.com/Waterboy1602/Addarr/commit/8ecd766a27199ad58a7a4e247cc5d9d5a337b49f).

**Comment by toddrob99** [comment]
I guess either I fixed this while I was working on other things, or it was only happening when I had additional logging in place for testing. Either way it seems to be a non-issue now.
